tests/unit/test_split_transfer.py

```diff
@@ -0,0 +1,54 @@
+import unittest
+from duneapi.types import Address
+
+from src.constants import COW_TOKEN_ADDRESS
+from src.fetch.period_slippage import SolverSlippage
+from src.fetch.transfer_file import Transfer, SplitTransfers, Overdraft
+from src.models import AccountingPeriod, Token
+from duneapi.types import Address
+from eth_typing import URI
+
+ONE_ETH = 10**18
+
+
+class TestSplitTransfers(unittest.TestCase):
+    def test_process_native_transfers(self):
+        class SplitTransfersTest(SplitTransfers):
+            def __init__(
+                self, period: AccountingPeriod, mixed_transfers: list[Transfer]
+            ):
+                SplitTransfers.__init__(self, period, mixed_transfers)
+
+            def process_native_transfers(
+                self, indexed_slippage: dict[Address, SolverSlippage]
+            ) -> int:
+                return SplitTransfers._process_native_transfers(self, indexed_slippage)
```

Comment:
@josojo - could you please tell me the reason you introduced a new class here? I don't see anything different about the constructor or process native transfer method (as they are both calling into the inherited implementation). I am working on writing more tests for this class and I am going to remove this if I can't determine the reason...

OH!!! Is it because `_process_native_transfers` is private! This shouldn't be a problem for CI because we don't validate the test directory. Similar to testing in Rust, you are able to write tests for private structs within the same file that it is defined... You're IDE will warn you that this is a private method. 

Anyway, I will try removing this in an upcoming PR and see how it goes if that is ok with you.

Reply:
sounds good